test_cases/Hottu 061/return

```diff
@@ -0,0 +1 @@
+0
```

Comment:
Program without prolog should fail with syntax error

Reply:
Opět záleží na implementaci